server/src/api/roster/roster-entity.ts

```diff
@@ -57,6 +57,11 @@ export abstract class RosterEntity extends BaseEntity {
   })
   lastName!: string;
 
+  @Column({
+    length: 20, default: 'Missing',
+  })
+  phoneNumber!: string;
```

Comment:
We're currently formatting phone numbers with database hooks as they're updated, so that they're saved in a consistently format. You can see an example of that here...

https://github.com/mystatus/covid19-reports/blob/b615a8e9677aeca714d5d25eb2b00e5af81c9ced/server/src/api/access-request/access-request.model.ts#L77

Reply:
done